packages/whale-api-client/__tests__/api/poolpairs.test.ts

```diff
@@ -615,14 +623,14 @@ describe('poolswap', () => {
         displaySymbol: 'DUSD'
       },
       apr: {
-        reward: 0,
-        total: 0.12174783188792529,
-        commission: 0.12174783188792529
+        reward: null,
+        total: null,
+        commission: null
```

Comment:
hmm.. can help verify what cause it `null` ?? :thinking:

Reply:
Hi, yup it's expected. `TEST-DUSD` is isolated in the graph, that's why it can't be computed and returns `null`. I kept the test as is and didn't add a path to test this scenario too

Reply:
Can also add DUSD-DFI pair so it will not be null, if preferred (?)

Reply:
If there's a path to USDT, then this can be computed 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/13292662/201007145-c5438166-fcb0-404b-a052-8a6d0708fa9f.png">

Reply:
For visualization 😄 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/13292662/201007424-55b5d596-66e4-4d26-a86f-5f24e030de46.png">